python/lsst/meas/deblender/plugins.py

```diff
@@ -748,10 +747,10 @@ def buildSymmetricTemplates(debResult, log, patchEdges=False, setOrigTemplate=Tr
 
 
 def rampFluxAtEdge(debResult, log, patchEdges=False):
-    """Adjust flux on the edges of the template footprints.
+    r"""Adjust flux on the edges of the template footprints.
```

Comment:
I've been able to get this to show an error either with or without the `r`. Let's compare notes next time we meet.